demisto_sdk/commands/lint/templates/dockerfile.jinja2

```
@@ -13,9 +13,13 @@ RUN chmod -R 775 /devwork
 {# Install requirments and missing deps #}
 RUN printf "{{ pypi_packs | join('\\n') }}" > /devwork/test-requirements.txt
 RUN OS_RELEASE=$(cat /etc/os-release); if echo "$OS_RELEASE" | grep -q "alpine"; \
-then apk add --no-cache --virtual .build-deps python3-dev gcc build-base; fi; \
+then apk add --no-cache --virtual .build-deps python3-dev gcc build-base; \
+elif echo "$OS_RELEASE" | grep -qi "Debian"; \
+then apt-get update && apt-get install -y --no-install-recommends gcc python3-dev;fi; \
 pip install --no-cache-dir -r /devwork/test-requirements.txt; \
-if echo "$OS_RELEASE" | grep -q "alpine"; then apk del .build-deps; fi;
+if echo "$OS_RELEASE" | grep -q "alpine"; then apk del .build-deps; \
+elif echo "$OS_RELEASE" | grep -qi "Debian"; \
+then apt-get autoremove -y gcc python3-dev;fi;
```

```suggestion
then apt-get purge -y --auto-remove gcc python3-dev; fi;
```

you should use `purge` from what I know